packages/deploy/deploy/800_oft_sand/802_oft_adapter_setup_sepolia.ts

```diff
@@ -0,0 +1,36 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {ethers} from 'ethers';
+import OFTSandBaseJson from '../../deployments/baseSepolia/OFTSand.json';
+import OFTSandBscJson from '../../deployments/bscTestnet/OFTSand.json';
+
```

Comment:
We may need to do something like this https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/packages/core/deploy/02_land/06_deploy_land_tunnel.ts#L30 to be able to read the json from other networks

Reply:
Unless we hardcode the OFTSand params that we need in the functions below